source/SIL.AppBuilder.BuildEngineApiClient.Tests/Integration/ProjectTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using Xunit;
+
+namespace SIL.AppBuilder.BuildEngineApiClient.Tests.Integration
+{
+    public class ProjectTests
+    {
+        // Note: You should change these values to match your environment.  These are not 
+        public string BaseUrl { get; set; } = "https://buildengine.gtis.guru";
+        public string ApiAccessKey { get; set; } = "BNLAVlQ7xlBoSqvRNbS153FqCWfcVCpX";
+        public string UserId { get; set; } = "chris_hubbard@sil.org";
+        public string GroupId { get; set; } = "LSDEV";
+        public string PublishingKey { get; set; } = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDik3IL0ldYovV6VYX3ICczNwJS2fl2fHkfaki/P4PWyl0p1glg/K5V74Vun9S3Fv+AwrMu8vUVXSuinINYqeFzIwc7M5r0QxAy60ExBZQiXYPs//maTJdyXkwZQkIQQNTf+NT6LBhNqv6iekzDO2AL/cirVq2wbuO8bzlBLLUc9uwLcXVd7IGZyJ9LLB5/T7gRPQAGif85m7lRAJ1R2+2QnPsBTMjyqcrKULSysLqd5+xXChljxSiJqVjGAh0jqpD+W7vo85iC9wqmq0H+MYTzE86qoXMh12k/boDIeZHBnkjRAZyRgq5uDvyVFk0c6Hn7fAGWvg04sWDr77yGNpIl hubbard@swd-hubbard-mac.local";
```

Comment:
did you intend to check this in? 

do you need to revoke access to things now? 

:astonished:

Reply:
It is a public key ... it is an example.  I didn't finish the comment ... sigh...